cmake/AddFdbTest.cmake

```diff
@@ -211,6 +211,7 @@ function(create_test_package)
       COMMENT "Package correctness archive"
       )
     add_custom_target(package_tests ALL DEPENDS ${tar_file})
+    add_custom_target(correctness_pkg ALL DEPENDS ${tar_file})
```

Comment:
It seems fine to add this if we expect `package_tests` may include other work unrelated to correctness in the future, and I also prefer the name to `package_tests`.

I suspect, though, that we'll need to add the dependencies in line 216 below to the `correctness_pkg` target. If we go this route, it probably makes sense to change 216 to add the dependencies to `correctness_pkg`, and then have `package_tests` depend on `correctness_pkg`.